chain/rosetta-rpc/src/utils.rs

```diff
@@ -269,14 +269,23 @@ where
     }
 }
 
+/// Zero-balance account (NEP-448)
+fn is_zero_balance_account(account: &near_primitives::account::Account) -> bool {
+    account.storage_usage() <= 770
```

Comment:
Please use `ZERO_BALANCE_ACCOUNT_STORAGE_LIMIT` instead of 770

Reply:
https://github.com/near/nearcore/pull/8833